src/main/java/eggventory/StockList.java

```diff
@@ -147,6 +147,40 @@ public String toString() {
         return ret;
     }
 
+    /**
+     * Prints every stock within stocklist whose stocktype matches query. Should only be called by Cli.
+     * @return The string of the stocklist whose stocktype matches query.
+     */
+    public String toString(String query) {
```

Comment:
toString should be the most generic print of the object and should never take params as far as i know.
also each stocktype name should be unique so i'm not sure what you're doing here. maybe this logic can be repurposed for the find function?

Reply:
wait is this an overload of toString...? i think it should be renamed

Reply:
ok sure then.